README.md

```diff
@@ -1,68 +1,51 @@
 
-### Setup
+##### Simulate simple, artificial lifeforms as they learn and evolve
 
-1. Install cmake & [Catch2](https://github.com/catchorg/Catch2) on your OS
-2. Clone this repository:  
-   HTTPS: `git clone https://github.com/Xerus2000/eve`  
-   SSH: `git clone git@github.com:Xerus2000/eve.git`
-3. Build 'eve' via CLion so that all tasks become available for running
-4. In CLion go to `Settings > Editor > File and Code Templates` and remove the "Created by" header in c/cpp files
-5. Execute this to set up the git commit message hooks:  
-   `git config core.hooksPath .dev/githooks`
+----
 
-### Structure
+#### Table of Content
 
-All production code goes into [src](src), separated into [model](src/model), [view](src/view) and [controller](src/controller).  
-[test](test) contains the tests and mirrors the source file structure.
+1. Setup
+2. Description
+3. Collaboration
+4. About
 
-### Classes and Inheritance
+----
 
-The folder structure should represent classes and inheritance.
+### Setup
 
-Therefore,
+* **Requirements:** Build tools: `cmake`, `make`, an ISO C++17 Compiler
+* **Download**: ``` git clone https://github.com/Xerus2000/eve```
+* **Build:** Run `cmake` and then `make`
+* **Run:** Start the simulation by executing `eve`
 
-- every class belongs in its own folder with the same name
-- every sub-class belongs within its parents folder
+----
 
-Thus, as of currently we have the following directory tree:
+### Description
 
-    object
-    ├── entity
-    │   ├── food
-    │   └── individual
-    │       ├── species1
-    │       ├── species2
-    │       └── ...
-    └── obstacle
+This project aims to simulate a two-dimensional environment with basic individuals; circles with a set of extremities (limbs, mouths, eyes). They shall eat randomly generated food and evolve through breeding. Throughout their life, individuals shall become smarter through reinforcement learning based on their own perception.
 
-Representing the following Inheritance Diagram:
+----
 
-            Object
-              │
-       +------+------+
-    Obstacle       Entity
-                     │
-              +------+------+
-              Food      Individual
-                            │
-                     +------+------+
-                    Sp1    Sp2    ...
+### Collaboration (through pull requests)
 
-## Testing
+* **Pull requests:** One reviewer required, always [squash and merge](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits)
 
-We use [Catch2](https://github.com/catchorg/Catch2) for testing.
+* **Commit guidelines:** see [Karma Runner](http://karma-runner.github.io/4.0/dev/git-commit-msg.html)
+  * `git config core.hooksPath .dev/githooks` for style enforcement
+* **Architecture:** Model-View-Controller
+  * Code is divided into `src/` and `test/`, with sub-folders `model/`, `view/` and `controller/` each
```

Comment:
I would formulate this differently to express that the test file structure resembles the src structure